File: DuDoong-Batch/src/main/java/band/gosrock/dto/SettlementPDFDto.java
package band.gosrock.dto;


import java.time.LocalDateTime;
import lombok.Builder;
import lombok.Getter;

@Builder
@Getter
public class SettlementPDFDto {
    private final String eventTitle;
    private final String hostName;
    private final LocalDateTime settlementAt;
    private final String dudoongTicketAmount;
    private final String pgTicketAmount;
    private final String totalAmount;
    private final String dudoongFee;
    private final String pgFee;
    private final String totalFee;
    private final String totalFeeVat;
    private final String totalSettlement;
    private final LocalDateTime now;
}

File: DuDoong-Batch/src/main/kotlin/band/gosrock/dto/SettlementPDFDto.kt
package band.gosrock.dto

import java.time.LocalDateTime

data class SettlementPDFDto(
    val eventTitle: String,
    val hostName: String,
    val settlementAt: LocalDateTime,
    val dudoongTicketAmount: String,
    val pgTicketAmount: String,
    val totalAmount: String,
    val dudoongFee: String,
    val pgFee: String,
    val totalFee: String,
    val totalFeeVat: String,
    val totalSettlement: String,
    val now: LocalDateTime,
)
